indexer-api/src/infra/api/v1/subscription/unshielded.rs

```diff
@@ -123,12 +128,32 @@ where
         .try_filter(move |event| ready(event.address == address_for_filter));
 
     let updates = try_stream! {
+        // Get the starting transaction ID (default to 1 for genesis if not specified)
+        let from_transaction_id = transaction_id.unwrap_or(1);
+
+        debug!(
+            address:?,
+            from_transaction_id;
+            "starting unshielded subscription from transaction ID"
```

Comment:
Remove obsolecte "from transaction ID", because that is obvious from the names field.